web/ui/src/common/miscUtils.js

```diff
@@ -207,6 +207,14 @@
                 return childCount;
             },
 
+            validateHostName: function(hostStr){
```

Comment:
did the rpc port also need to be >1024?  I know those up to 1024 are priviledged ports that can only be bound by root, but not sure if the rpcport is setup as root or not